src/connector/src/sink/catalog/mod.rs

```diff
@@ -166,6 +166,31 @@ impl SinkFormatDesc {
         }))
     }
 
+    #[cfg(feature = "sink_bench")]
```

Comment:
What's the purpose for this mock method?

Reply:
This method is to be compatible with sink's two syntaxes, and our redis is only support format syntax, so there is no redis in the method, but the mock needs to be converted to format by redis's option, I will move this method to the bench directory.